PipelinesTasks/PRMetrics/buildAndReleaseTask/Updaters/CodeMetrics.ts

```diff
@@ -0,0 +1,429 @@
+/* eslint-disable spaced-comment */
+/* eslint-disable no-array-constructor */
+/* eslint-disable space-before-function-paren */
+/* eslint-disable indent */
+/* eslint-disable semi */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IMetrics } from './iMetrics';
+import ProcessWrapper from '../wrappers/ProcessWrapper';
+import TaskLibWrapper from '../wrappers/taskLibWrapper';
+import { isNullOrWhitespace } from './CodeMetricsHelpers';
+
+class CodeMetrics {
+    public Size: string;
+    public Metrics: IMetrics; // was [Hashtable]
+    public IgnoredFilesWithLinesAdded: Array<string>; // was [System.Collections.Generic.List[string]]
+    public IgnoredFilesWithoutLinesAdded: Array<string>; // was [System.Collections.Generic.List[string]]
+    public BaseSize: number;
+    public ExpectedTestCode: number;
```

Comment:
Would be better not to expose implementation details. Can you wrap these in a getter instead?